proposals/acp-specification/index.bs

```diff
@@ -3,15 +3,41 @@ Title: Access Control Policies
 Shortname: acp
 Level: 1
 Max ToC Depth: 2
+Boilerplate: style-darkmode off
 Status: w3c/ED
 Group: solid-cg
 URL: https://solid.github.io/authorization-panel/acp-specification
 Editor: Aaron Coburn, Inrupt, https://inrupt.com
 Editor: Justin Bingham, Janeiro Digital, https://janeirodigital.com
 Editor: Matthieu Bosquet, Inrupt, https://inrupt.com 
-Abstract: Access Control Policies are a mechanism to define and enforce 
-  authorization to resources stored in a Solid Pod.
-Boilerplate: style-darkmode off
+Abstract:
+  This document defines the Access Control Policies Language (ACP), a language for
+  enforcing resource access against a set of conditions. These conditions are provided
+  as Access Controls, Policies and Matchers expressed in the form of an RDF graph.
+  RDF graphs that are used in this manner are called "authorization graphs" in ACP.
```

Comment:
The content starting at "RDF graphs..." feels like it would be better in an introductory section as its transitioned from a high-level statement of what it is, into an explanation of terms/concepts.